fe/fe-core/src/main/java/com/starrocks/common/InternalErrorCode.java

```diff
@@ -38,7 +38,8 @@ public enum InternalErrorCode {
     TOO_MANY_FAILURE_ROWS_ERR(102),
     CREATE_TASKS_ERR(103),
     TASKS_ABORT_ERR(104),
-    SLOW_RUNNING_ERR(105);
+    SLOW_RUNNING_ERR(105),
+    PARSE_ERR(106);
```

Comment:
Don't use internal error code, should use ErrorCode

Reply:
Use DATA_FORMAT_PARSE_ERROR?
The word `PARSE_ERR` is to general.

Reply:
@HangyuanLiu fixed